Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -528,7 +522,7 @@ - (void)textInputDidChange
 
   if (_onChange) {
     _onChange(@{
-       @"text": [self.attributedText.string copy], // [TODO(macOS Candidate ISS#2710739)
```

Comment:
Was this macOS change pushed upstream and that's why we're removing our TODO?

Reply:
Yes, with 15dda0ab5a491dcc83539f9ef32c9896be41074a.